app/views/pair_requests/index.html.erb

```diff
@@ -27,7 +27,7 @@
         <div class="table-row odd:bg-white even:bg-neutral-100 hover:bg-neutral-200 divide-x-2 divide-black">
 
           <%= render TableCellComponent.new(container_link: pair_request) do %>
-            <%= pair_request.partner_for(current_user).email %>
+            <p>John</p><p>Doe</p>
```

Comment:
probably should not use the John Doe placeholder

Reply:
use email for now